scheduler/src/cook/mesos/api.clj

```diff
@@ -231,9 +241,25 @@
                 db uuid))
     (throw (ex-info (str "UUID " uuid " already used") {:uuid uuid}))))
 
+
+(defn job-owner-by-uuid
+  "Returns the username that owns the given UUID, or nil if no job with
+  that UUID exists."
+  [db uuid]
+  (let [uuid (if (string? uuid)
+               (UUID/fromString uuid)
+               uuid)]
+    (some-> (d/q '[:find ?owner
```

Comment:
Sure, if you want. Since they are equivalent forms and the comment was ambiguous, I took that as more general information rather than a change request.